content/en/kanvas/reference/help-and-support.md

```
@@ -0,0 +1,42 @@
+description: |
+  How Kanvas submits support requests, and the two preconditions - a remote provider and a signed-in identity - that Help and Support requires.
+weight: 20
+---
+
+Kanvas offers a **Help and Support** form for sending a support request to the Layer5 team. You can open it from the Help menu, and from the **Get Help** action in Kanvas's troubleshooting panel.
+
+## How a support request is delivered
+
+A support request never leaves your browser for a third party. Kanvas submits the form to Meshery Server, which forwards it to your provider's support intake:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)